core/sr-api-macros/Cargo.toml

```diff
@@ -25,6 +25,7 @@ criterion = "0.2"
 consensus_common = { package = "substrate-consensus-common", path = "../consensus/common" }
 codec = { package = "parity-scale-codec", version = "1.0.0" }
 trybuild = "1.0"
+protobuf = "~2.8.0"
```

Comment:
Locked as protobuf broke semver and breaks libp2p-core build. See https://github.com/paritytech/substrate/pull/3723/files